charts/rabbitmq/Chart.yaml

```diff
@@ -3,7 +3,7 @@ name: rabbitmq
 description: A central RabbitMQ message broker
 type: application
 
-version: 0.1.1
+version: 0.2.1
```

Comment:
Surely `0.2.0`?

Reply:
Thanks for the catch 👍 not sure what I was thinking there 🤦‍♂️